### PR TITLE
Changed double-quotes to single-quotes

### PR DESCRIPTION
**Pull request by softwarespot**

### CODE REVIEW

**Comment by softwarespot** [comment]
I have rebased the following PR

File: README.md [hunk 1]
```diff
@@ -674,7 +675,16 @@ For a complete replacement with namespace and delegation, refer to https://githu
   $(document).ready(eventHandler);
 
   // Native
-  document.addEventListener("DOMContentLoaded", eventHandler);
+<<<<<<< HEAD
```
Comment (camsong) [inline README.md]: Conflicts should be fixed

File: README.md [hunk 2]
```diff
@@ -956,8 +971,12 @@ Most of utilities are found by native API. Others advanced functions could be ch
   $.makeArray(arrayLike);
 
   // Native
+<<<<<<< HEAD
+  Array.prototype.slice.call(arrayLike);
+=======
   [].slice.call(arrayLike);
-  
+>>>>>>> origin/single-quotes
```
Comment (camsong) [inline README.md]: Conflicts

**Comment by camsong** [comment]
Thanks for you contribution, can you squash all this into one commit?

**Pull request closed without merging**